identity-server/templates/templates/IdentityServerAspNetIdentity/.template.config/template.json

```
@@ -2,12 +2,13 @@
   "author": "Duende Software",
   "classifications": [
     "Web",
+    "Duende",
     "IdentityServer"
   ],
   "name": "Duende IdentityServer with ASP.NET Core Identity",
-  "identity": "IdentityServer.AspNetIdentity",
-  "groupIdentity": "IdentityServer.AspNetIdentity",
-  "shortName": "isaspid",
+  "identity": "Duende.IdentityServer.AspNetIdentity",
+  "groupIdentity": "Duende.IdentityServer.AspNetIdentity",
+  "shortName": "is-asp-id",
```

The new naming convention that you've adopted is definitely better, but the downside is that old documentation and other content (3rd party blogs, pluralsight content, maybe even our own youtube tutorials) all refer to the old names. I think we should add the new names, but not break the old ones:
```suggestion
  "shortName": ["is-asp-id", "isaspid"],
```

---

Agreed, did this. Although we'll still break them via the names of the actual packages.